crates/language_server/src/convert.rs

```diff
@@ -102,19 +103,18 @@ pub(crate) mod diag {
     }
 
     impl IntoLspDiagnostic<'_> for &LoadingProblem<'_> {
-        type Feed = ();
-
-        fn into_lsp_diagnostic(self, _feed: &()) -> Option<Diagnostic> {
-            let range = Range {
-                start: Position {
-                    line: 0,
-                    character: 0,
-                },
-                end: Position {
-                    line: 0,
-                    character: 1,
-                },
-            };
+        type Feed = LineInfo;
+
+        fn into_lsp_diagnostic(self, line_info: &LineInfo) -> Option<Diagnostic> {
+            let range = self
+                .get_region()
+                .unwrap_or(Region::new(
+                    roc_region::all::Position::new(0),
+                    roc_region::all::Position::new(10000000),
+                ))
+                .to_range(line_info);
+
+            debug!("range is {:?}", range);
```

Comment:
Presumably we either want to remove these `debug!`s or give more context to whoever would be reading them in the LSP logs.